providers/shared/components/firewall/id.ftl

```diff
@@ -75,7 +75,7 @@
                         ]
                     }
                 ]
-            }
+            },
```

Comment:
This might have been fixed? maybe need to rebase your PR?

Reply:
The comma is still missing on master